src/assets/stylesheets/Instructions.scss

```diff
@@ -269,3 +275,44 @@
     margin: 0;
   }
 }
+
+#app,
+#wc {
+  .c-instruction-tabs {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .react-tabs {
+      border: 1px solid var(--editor-color-outline);
+
+      .react-tabs__tab-list {
+        border-bottom: 1px solid var(--editor-color-outline);
+      }
+    }
+
+    .react-tabs__tab {
+      background: var(--rpf-off-white);
+      padding-inline: var(--space-1-5);
+
+      &--selected {
+        background: var(--rpf-white);
+      }
+    }
+
+    .react-tabs__tab-panel {
+      .project-instructions {
+        padding-inline: var(--space-1);
+      }
+    }
+
+    textarea {
+      @include typography.style-1();
+
+      border: none;
+      height: 100%;
+      overflow-y: scroll;
+      padding: var(--space-1);
+    }
+  }
```

Comment:
I think some of these can change to logical properties, e.g. `height -> block-size` and `border-bottom -> border-block-end`